### ISSUE TITLE
Extension not honoring local eslint configuration

### ISSUE THREAD

**Issue by sathomas** [apexskier/nova-eslint#40]
The extension is reporting vue errors even though local configuration has set those rules to be off

**To reproduce**

1. Disable a vue ESLint rule in `eslintrc.yml`
2. Create a `.vue` file with code that violates that rule.

**Expected behavior**

No warnings or errors.

**Screenshots**

![Screen-Bk7lLOSm](https://user-images.githubusercontent.com/1731910/101659557-3487bd00-3a14-11eb-934c-b0bd2644fa9a.png)

![Screen-2M9Rw559](https://user-images.githubusercontent.com/1731910/101659568-381b4400-3a14-11eb-994a-ed32a34795b9.png)

**Environment**

- Nova version: 3.1 (221256)
- Extension version: 1.5.2
- ESLint version: v7.7.0
- macOS version: 11.0.1
- NodeJS information:
  - node version: v14.15.1
  - npm version: 6.14.8
  - installation method: homebrew

**Extension console output**

```
ESLint[11:51:47.410000] activating...
ESLint[11:51:47.832000] activated
```

**Additional context**

Running `eslint` from command line shows no errors or warnings

```
> eslint public/src/components/*.vue "gulp/**/*.js" "test/**/*.js" common/*.js "app/**/*.js" "public/src/**/*.js" "rpcServices/*.js"


```

**Comment by FlorianWendelborn** [comment]
I’ve experienced similar behavior with VSCode where some of the Vue rules would always run, regardless of settings... just another perk of being in the Vue "let’s invent our own file format" ecosystem. 🙁

**Comment by apexskier** [comment]
@sathomas What's the output of running `./node_modules/.bin/eslint --print-config ./path/to/file.vue`?

**Comment by apexskier** [comment]
Never mind, I have a local reproduction.

**Comment by apexskier** [comment]
Figured it out. The eslint extension is not causing the issue, it's [nova-vue](https://github.com/tommasongr/nova-vue), as I should have noticed in your screenshot. I'm going to close this here, and you should open a new issue over there.

I verified by running the exact same command this extension runs manually and didn't see the error reported in the CLI.

![101659568-381b4400-3a14-11eb-994a-ed32a34795b9](https://user-images.githubusercontent.com/329222/105010976-ad7b2b00-5a3c-11eb-9a8e-c612ce010c8c.png)
